app/src/main/java/com/jerboa/db/entity/AppSettings.kt

```diff
@@ -124,4 +125,9 @@ data class AppSettings(
         defaultValue = "1",
     )
     val swipeToActionPreset: Int,
+    @ColumnInfo(
+        name = "last_version_code_viewed",
+        defaultValue = "0",
+    )
+    val lastVersionCodeViewed: Int,
```

Comment:
I would probably have made a separate table for this. It's not really a app setting. But this is fine